_posts/2020-09-21-fall-2020-meeting-announced.markdown

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: "PyHC Fall 2020 Meeting Dates Announced."
+author: jibarnum
+---
+
+PyHC serves as a community knowledge base for performing heliophysics research in Python, aiming to provide a variety of tutorials, resources, a list of useful packages, general discussion, and advice. Twice yearly, PyHC meets in person. These meetings are open to anyone from individuals already utilizing Python heavily in developing open-source heliophysics packages, as well as those who are just getting into using Python and open source development. 
+
+**PyHC's second meeting of 2020 will be held remotely on Zoom over a four-week long period, starting Monday, October 26th, 2020 and ending Monday, November 16th, 2020.** We’ll hold tutorial, hackathon, and unconference sessions, as well as hold a few plenaries during which we'll have presentations on topics relevant to the community. See [the full agenda](https://docs.google.com/spreadsheets/d/18lrxi8XsjDZ76MjWZPFOovu1JWrdjvoLZmyZfS_LZ3A/edit#gid=0) for exact dates/times of the various sessions. 
+The meeting's web page will be up shortly. **Registration, which is free, but required, can be found [here](https://forms.gle/ZYESjbYwiGooCkKJ7).** The meeting's web page will also have a link to registration, as well as the Zoom telecon information. If you’d like to join our mailing list to learn more information about the meeting and other upcoming events, please see the [PyHC Contact page](http://heliopython.org/contact/) for instructions. We also have a [Riot chat group]( https://riot.im/app/#/room/#heliopython:openastronomy.org
+) where we discuss various PyHC issues/topics, as well as discussing other Python-related questions.
+
+**Registration Deadline: COB, Friday, October 23rd, 2020.**  
+Note: If you already registered and later decide to change your registration, you should be able to do that directly, but if you experience any difficulties please email [Julie Barnum](mailto:Julie.Barnum@lasp.colorado.edu) or [Shawn Polson](shpo9723@colorado.edu).
```

Comment:
My email should have `mailto:` prepended to it like yours. And I'd prefer my LASP email here: shawn.polson@lasp.colorado.edu

Reply:
Good catch, I'll fix that.